src/transaction/log_manager.c

```diff
@@ -4780,7 +4781,6 @@ log_commit_local (THREAD_ENTRY * thread_p, LOG_TDES * tdes, bool retain_lock, bo
   LSA_SET_NULL (&tdes->undo_nxlsa);
 
   /* destroy all transaction's remaining temporary files */
-  file_tempcache_drop_tran_temp_files (thread_p);
```

Comment:
Is it ok to remove it? it seems to drop all temporary files including created filed in query execution.

Reply:
We need to remove the all files excluding qlist file (result file), but exact processing can be by storage manager.
I remain this processing to storage manager (in charge of develop team 2).
Currently the temp files are removed at server stop.

Reply:
I found a way not to remove this code. When result file is created we can set the file to preserve type. I will push the code.